{"text": "@WHO @WHOWPRO @WHOSEARO @WHO_Europe @pahowho @WHOEMRO @WHOAFRO Thanks for notification, try @moayush for remedy \n\n#coronavirus #CoronaOutbreak", "id": "1222004557380612097", "context_annotations": [{"domain": {"id": "123", "name": "Ongoing News Story", "description": "Ongoing News Stories like 'Brexit'"}, "entity": {"id": "1220701888179359745", "name": "COVID-19"}}, {"domain": {"id": "29", "name": "Events [Entity Service]", "description": "Entity Service related Events domain"}, "entity": {"id": "1219322342091788288", "name": "The WHO is holding an emergency meeting on the new coronavirus", "description": "According to the announcement, the World Health Organization's Emergency Committee \"will ascertain whether the outbreak constitutes a public health emergency of international concern\" and provide recommendations on how to manage the situation."}}, {"domain": {"id": "29", "name": "Events [Entity Service]", "description": "Entity Service related Events domain"}, "entity": {"id": "1220416677990420485", "name": "'Too early' to declare coronavirus outbreak as global health emergency, says WHO", "description": "The World Health Organization declined for a second day to declare the pneumonia-like coronavirus a global health emergency. China confirmed that the death toll has reached 41 and that 1000 people have been infected."}}], "reply_settings": "everyone", "conversation_id": "1221800661039419398", "entities": {"mentions": [{"start": 0, "end": 4, "username": "WHO", "id": "14499829"}, {"start": 5, "end": 13, "username": "WHOWPRO", "id": "3794682452"}, {"start": 14, "end": 23, "username": "WHOSEARO", "id": "1545915336"}, {"start": 24, "end": 35, "username": "WHO_Europe", "id": "125403342"}, {"start": 36, "end": 44, "username": "pahowho", "id": "20629893"}, {"start": 45, "end": 53, "username": "WHOEMRO", "id": "259372802"}, {"start": 54, "end": 62, "username": "WHOAFRO", "id": "544389588"}, {"start": 92, "end": 100, "username": "moayush", "id": "2650295678"}], "hashtags": [{"start": 114, "end": 126, "tag": "coronavirus"}, {"start": 127, "end": 142, "tag": "CoronaOutbreak"}]}, "public_metrics": {"retweet_count": 0, "reply_count": 0, "like_count": 1, "quote_count": 0}, "author_id": "37341212", "source": "Twitter Web App", "referenced_tweets": [{"type": "replied_to", "id": "1221800661039419398"}], "possibly_sensitive": false, "lang": "en", "created_at": "2020-01-28T03:52:44.000Z", "in_reply_to_user_id": "14499829"}